libraries/mongodb.rb

```diff
@@ -49,7 +49,7 @@ def self.configure_replicaset(node, name, members)
     end
     
     # Want the node originating the connection to be included in the replicaset
-    members << node unless members.any? {|m| m.name == node.name }
+    members << node unless members.any? {|m| m.name == node['name'] }
```

Comment:
will members.any stay the same or will the format for it be changed as well here?

Reply:
right that makes sense.